swarm.cabal

```diff
@@ -152,6 +153,7 @@ library
                       minimorph                     >= 0.3 && < 0.4,
                       mtl                           >= 2.2.2 && < 2.3,
                       murmur3                       >= 1.0.4 && < 1.1,
+                      natural-sort,
```

Comment:
How to determine appropriate bounds for this package?

Reply:
Hmm, well, it was uploaded in 2012 and has never been updated since (and probably never will).  Since it is so old, there's no reason to allow anything but the latest version.  We might as well just put `>= 0.1.2 && < 0.2`.